### PR TITLE
move CHW endpoints to a new the new path: /unimplemented

### PR DESCRIPTION
**Pull request by MarcFontaine**
## Description
This PR moves the CHW endpoints to a new path: /unimplemented.
New CHW features can be added to the /unimplemented endpoint with any danger of
interference with api/V1 features man later be moved to api/V1.

## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [X] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [X] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: wallet-new/src/Cardano/Wallet/API.hs [hunk 1]
```diff
@@ -54,8 +56,9 @@ type InternalAPI = "api" :> "internal" :> Internal.API
 internalAPI :: Proxy InternalAPI
 internalAPI = Proxy
 
+type WIPAPI = "api" :> "unimplemented" :> WIP.API
```
Comment (denisshevchenko) [inline wallet-new/src/Cardano/Wallet/API.hs]: If the path is `unimplemented` - why did you named it `WIPAPI`? Why not `UnimplementedAPI`? :-) Is it too long?

File: wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+-- | WIP @Servant@ handlers the are not part of the offical api yet.
+handlers :: PassiveWalletLayer IO -> ServerT WIP.API Handler
+handlers pwl = checkExternalWallet pwl
+           :<|> newExternalWallet pwl
+           :<|> deleteExternalWallet pwl
+
+-- | Creates a new or restores an existing @wallet@ given a 'NewWallet' payload.
+-- Returns to the client the representation of the created or restored
+-- wallet in the 'Wallet' type.
+newWallet :: PassiveWalletLayer IO
```
Comment (denisshevchenko) [inline wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs]: Hm... I see our unimplemented endpoints in `wallet-new/src/Cardano/Wallet/API/WIP.hs`, and handlers for them here (`checkExternalWallet`, `newExternalWallet` and `deleteExternalWallet`), but why did you put `newWallet` handler here? Is it unimplemented handler too??

File: wallet-new/src/Cardano/Wallet/API.hs (hunk 1 above)
Comment (MarcFontaine) [reply wallet-new/src/Cardano/Wallet/API.hs]: To me its sounds strange to name a function `unimplemented` if it is actually at least somehow implemented.
I have no strong opinion about how to name the endpoint itself
Alfredo suggested to use `unimplemented` to make clear that is not meant for end users.

File: wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs (hunk 2 above)
Comment (MarcFontaine) [reply wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs]: ups
will fix it

Comment (MarcFontaine) [reply wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs]: Thanks for spotting..I made a forced-update to fix this.

**Pull request merged**